src/panels/lovelace/editor/config-elements/hui-calendar-card-editor.ts

```diff
@@ -45,10 +45,16 @@ export class HuiCalendarCardEditor extends LitElement
 
   @internalProperty() private _configEntities?: string[];
 
-  public setConfig(config: CalendarCardConfig): void {
+  @internalProperty() private _options?: Record<string, any>;
+
+  public setConfig(
```

Comment:
I don't think we should use the `setConfig` to set the options, we should just use a property for that.